WordPress/gutenberg
# Add support for align wide to deprecated versions of gallery block

## Description
Fixes #18417, which identified that gallery blocks don't properly retain their `align` attribute when upgrading from a deprecation.

To fix this I've added `supports: { align: true }` to all of the deprecations for the gallery.

## How has this been tested?
Updated deprecation test fixtures to also test the align attribute.

1. Paste this older version of the gallery block into the Code Editor view (note it has wide alignment attribute set correctly).
```

<figure class="wp-block-gallery columns-3 is-cropped alignwide">
	<ul class="blocks-gallery-grid">
		<li class="blocks-gallery-item">
			<figure>
				<img src="https://sergioestevaofolio.files.wordpress.com/2016/09/cropped-img_9054-1.jpg?w=190" alt="" data-id="1421" class="wp-image-1421"/>
			</figure>
		</li>
		<li class="blocks-gallery-item">
			<figure>
				<img src="https://sergioestevaofolio.files.wordpress.com/2017/09/cropped-l1001498-1.jpg?w=580" alt="" data-id="1440" class="wp-image-1440"/>
			</figure>
		</li>
		<li class="blocks-gallery-item">
			<figure>
				<img src="https://sergioestevaofolio.files.wordpress.com/2017/05/cropped-l1005945-2-2.jpg?w=580" alt="" data-id="1362" class="wp-image-1362"/>
			</figure>
		</li>
	</ul>
</figure>

```
2. Switch to the Visual Editor
3. Observe that the new gallery block has the alignment set correctly when viewing the toolbar option:
<img width="325" alt="Screen Shot 2020-01-09 at 3 53 22 pm" src="https://user-images.githubusercontent.com/677833/72048691-50607b80-32f8-11ea-8084-af83031c0f4c.png">

## Types of changes

Bug fix (non-breaking change which fixes an issue)



## Checklist:
